source/index.html.md

```diff
@@ -1431,34 +1431,35 @@ GET https://gateway.clearhaus.com/account
-           |  40420 |  Merchant blocked by cardholder
-           |  50000 |  Clearhaus error
+When you make a new transaction the JSON response includes one of the status
+codes in the following table. The rightmost columns indicate which transaction
+types a particular status code can be received for; Au, Ca, Re, Vo and Cr are
+short for authorizations, captures, refunds, voids and credits, respectively.
+
+Status     | `code` |  Meaning                            | Au      | Ca      | Re      | Vo      | Cr      |
+---------- | ------ | ----------------------------------- | ------- | ------- | ------- | ------- | ------- |
+Approved   |  20000 |  Approved                           | &check; | &check; | &check; | &check; | &check; |
+Declined   |  40000 |  General input error                | &check; | &check; | &check; | &check; | &check; |
+           |  40110 |  Invalid card number                | &check; |         |         |         | &check; |
+           |  40111 |  Unsupported card scheme            | &check; |         |         |         | &check; |
+           |  40120 |  Invalid CSC                        | &check; |         |         |         |         |
+           |  40130 |  Invalid expire date                | &check; |         |         |         | &check; |
+           |  40135 |  Card expired                       | &check; | &check; | &check; | &check; | &check; |
+           |  40140 |  Invalid currency                   | &check; |         |         |         | &check; |
+           |  40150 |  Invalid text on statement          | &check; | &check; | &check; |         |         |
+           |  40200 |  Clearhaus rule violation           | &check; | &check; | &check; | &check; | &check; |
+           |  40300 |  3-D Secure problem                 | &check; |         |         |         |         |
+           |  40310 |  3-D Secure authentication failure  | &check; |         |         |         |         |
+           |  40400 |  Backend problem                    | &check; |         |         | &check; | &check; |
+           |  40410 |  Declined by issuer or card scheme  | &check; |         |         | &check; | &check; |
+           |  40411 |  Card restricted                    | &check; |         |         |         | &check; |
+           |  40412 |  Card lost or stolen                | &check; |         |         |         | &check; |
+           |  40413 |  Insufficient funds                 | &check; |         |         |         | &check; |
+           |  40414 |  Suspected fraud                    | &check; |         |         |         | &check; |
+           |  40415 |  Amount limit exceeded              | &check; |         |         |         | &check; |
+           |  40416 |  Additional authentication required | &check; |         |         |         | &check; |
+           |  40420 |  Merchant blocked by cardholder     | &check; |         |         |         |         |
```

Comment:
The code isn't used currently but if we start using it, it will also be possible for credits. It makes sense to include credits, I think.

Fixed by https://github.com/clearhaus/gateway-api-docs/pull/134/commits/0ab11948fcc280ef7baaa5e8f7ab9fdad6bfdccb.